.github/workflows/dependabot_pr.yaml

```
@@ -0,0 +1,12 @@
+name: "Dependabot Auto Approve and Merge"
+
+on:
+  pull_request:
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  auto-merge:
+    uses: canonical/sdcore-github-workflows/.github/workflows/dependabot_pr.yaml@main
```

```suggestion
    uses: canonical/sdcore-github-workflows/.github/workflows/dependabot_pr.yaml@main
    secrets: inherit
```
The remote workflow uses `GITHUB_TOKEN`, which needs to be inherited from the calling job.

---

Does it need this? For example, the publish-charm workflow also uses the `GITHUB_TOKEN` secret, and it does not get inherited explicitly. I think there's something different about those default GitHub secrets.

---

Yeah, looks like you're right. Sorry for confusion.